src/main/java/com/efficio/fieldbook/web/fieldmap/controller/FieldmapController.java

```diff
@@ -731,4 +736,19 @@ public String getProgramLocation() {
 	public String getProgramID() {
 		return this.getCurrentProjectId();
 	}
+
+	private void validateFieldMapDataSetInfo(final List<FieldMapDatasetInfo> datasetList, final Map<String, Boolean> validatedFieldMapDatasetInfo) {
```

Comment:
Make return type boolean, no need for map parameter. Immediately return if invalid value found

Reply:
Please create unit test for this method